docs/download.md

```diff
@@ -4,44 +4,60 @@ title: Download
 sidebar: nav.html
 ---
 
-{% include anchor.html class="h3" title="File Download" hash="file" %}
+{% include anchor.html class="h3" title="Download" hash="file" %}
 
-The PouchDB file should come before any files that use it.
-
-For use in an HTML file:
+Quick start:
```

Comment:
I haven't seen many people (even beginners) get confused about this, so I don't think it's worth mentioning. Instead I just say "quick start."